sdks/python/apache_beam/runners/portability/portable_stager_test.py

```diff
@@ -48,7 +48,10 @@ def tearDown(self):
     if self._remote_dir:
       shutil.rmtree(self._remote_dir)
 
-  def stage_files(self, files):
+  def _stage_files(self, files):
+    """
```

Comment:
The reason I asked for docstring is that it would be useful to describe the structure of `files` since it is not obvious without reading the method. How about:

```
Utility method to stage files.

Args: 

  files: a list of tuples of the form [(local_name, remote_name),...] describing the name of the artifacts in local temp folder and desired name in staging location. 
```

Reply:
Made the change